internal/provider/resource_tsuru_service_instance_bind.go

```diff
@@ -115,7 +139,10 @@ func resourceTsuruServiceInstanceBindRead(ctx context.Context, d *schema.Resourc
 	}
 	service := parts[0]
 	instanceName := parts[1]
-	app := parts[2]
+	name := parts[2]
+	if name == "tsuru-job" {
```

Comment:
I would hope nobody tries to create a simple app called `tsuru-job`.
But in order to prevent this corner case, I would change this to 
`if len(parts)==4 ...`

Reply:
but i wonder.... why the `IDtoParts()` does not error when the expected is greater... 🤔 
well, it works

Reply:
because IDtoParts only checks if the length is lower than the expected value 🙈